### ISSUE TITLE
NullPointerException: Attempt to invoke virtual method 'int android.content.Context.getColor(int)' on a null object reference

### ISSUE THREAD

**Issue by galadril** [domoticz/domoticz-android#153]
```bash
java.lang.NullPointerException: Attempt to invoke virtual method 'int android.content.Context.getColor(int)' on a null object reference
	at com.afollestad.materialdialogs.util.DialogUtils.getColor(DialogUtils.java:115)
	at com.afollestad.materialdialogs.MaterialDialog$Builder.&lt;init&gt;(MaterialDialog.java:477)
	at nl.hnogames.domoticz.UI.GraphDialog.&lt;init&gt;(GraphDialog.java:63)
	at nl.hnogames.domoticz.Fragments.Temperature$6.onReceive(Temperature.java:217)
	at nl.hnogames.domoticz.Domoticz.GraphDataParser.parseResult(GraphDataParser.java:63)
	at nl.hnogames.domoticz.Utils.RequestUtil$7.onResponse(RequestUtil.java:207)
	at nl.hnogames.domoticz.Utils.RequestUtil$7.onResponse(RequestUtil.java:197)
	at com.android.volley.toolbox.JsonRequest.deliverResponse(JsonRequest.java:65)
	at com.android.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:99)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:5417)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)

```

**Comment by galadril** [comment]
we need to rework the whole graph dialog to an activity..

**Comment by RoadXY** [comment]
Agreed. I'll create an issue with feature request for it.

**Comment by RoadXY** [comment]
Will be solved by issue #216
